site/app/templates/forum/ForumBar.twig

```diff
@@ -158,6 +158,27 @@
 		</div>
 	{% endif %}
 </div>
+<div class="row mt-2">
```

Comment:
The placement of this button might add too much clutter to the forum bar and more UI discussion will be needed. I might suggest putting this button in the filter dropdown bar, but this feature is not necessarily a 'filter' or moving this button inline with the forum bar.